Source/WebCore/editing/MarkupAccumulator.cpp

```diff
@@ -466,7 +466,7 @@ void MarkupAccumulator::generateUniquePrefix(QualifiedName& prefixedName, const
     AtomString name;
     do {
         // FIXME: We should create makeAtomString, which would be more efficient.
-        name = makeString("NS", ++m_prefixLevel);
+        name = makeAtomString("NS", ++m_prefixLevel);
```

Comment:
You added `_s` above where switching from `makeString` to `makeAtomString` but not here. Should be consistent within the same patch. I suppose given the direction you have been going, eventually we will change `StringConcatenate` to require `ASCIILiteral` or `StringView` rather than having it work on `const char*`.